extensions/amp-timeago/0.1/validator-amp-timeago.protoascii

```diff
@@ -0,0 +1,71 @@
+#
+# Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-timeago
+  html_format: AMP
+  tag_name: "SCRIPT"
+  spec_name: "amp-timeago extension .js script"
+  satisfies: "amp-timeago extension .js script"
+  requires: "amp-timeago"
+  mandatory_parent: "HEAD"
+  unique: true
+  attrs: {
+    name: "async"
+    mandatory: true
+    value: ""
+  }
+  attrs: {
+    name: "custom-element"
+    mandatory: true
+    value: "amp-timeago"
+    dispatch_key: true
+  }
+  attrs: { name: "nonce" }
+  attrs: {
+    name: "src"
+    mandatory: true
+    value_regex: "https://cdn\.ampproject\.org/v0/amp-timeago-(latest|0\.1).js"
+  }
+  attrs: {
+    name: "type"
+    value: "text/javascript"
+  }
+  cdata: {
+    blacklisted_cdata_regex: {
+      regex: "."
+      error_message: "contents"
+    }
+  }
+  spec_url: "https://www.ampproject.org/docs/reference/components/amp-timeago"
+}
+tags: {  # <amp-timeago>
+  html_format: AMP
+  tag_name: "AMP-TIMEAGO"
+  satisfies: "amp-timeago"
+  requires: "amp-timeago extension .js script"
+  attrs: { name: "datetime" }
+  attrs: { name: "locale" }
```

Comment:
Similarly the value of `locale` can also be enforced to a set of languages, though you may not want to have to update this for each new language added/supported.

```
attrs: {
  name: "locale"
  value_regex: "ar|be|bg|ca|..."
}
```